app/modules/project/routes/project-service.ts

```diff
@@ -5,22 +5,22 @@ export class ProjectService {
     constructor() {}
 
     public create = async (reqBody: Object): Promise<ProjectInteface> => {
-        return await projectModel.create(reqBody);
+        return projectModel.create(reqBody);
     };
 
     public get = async (): Promise<ProjectInteface[]> => {
-        return await projectModel.find({});
+        return projectModel.find({});
     };
 
     public getById = async (projectID: string): Promise<ProjectInteface[]> => {
-        return await projectModel.find({ ID: projectID });
+        return projectModel.find({ ID: projectID });
```

Comment:
You could use `findOne` and return type can be `Promise<ProjectInterface>`

Reply:
One more thing, conventionally interfaces starts with `I` meaning `ProjectInterface` could become `IProject` which represents the domain, and based on that, each and every other entity would be created, such as Mongoose schema.